proxies/live/apiproxy/targets/target.xml

```diff
@@ -19,6 +49,12 @@
       </Step>
       <Condition>oauthV2.OauthV2.VerifyAccessToken.failed</Condition>
     </FaultRule>
+    <FaultRule name="gpc_access_token_expired">
+     <Step>
+         <Name>AssignMessage.GPCAuthErrorResponse</Name>
+      </Step>
+            <Condition>GPCPFSAuthResponse.status.code=="500"</Condition>
```

Comment:
Change status code to 403, AWS Authorizer Lambda will only ever return 200 or 403 depending on wether the token is valid or invalid.